### PR TITLE
Use `base-with-services` compose and install `aiida-quantumespresso`

### PR DESCRIPTION
**Pull request by mbercx**
Note that this should definitely not be merged, I'm only opening the PR to run the CI workflow,

### CODE REVIEW

**Comment by csadorf** [comment]
@mbercx If you keep it in draft mode it can't be merged, but the CI will still run.

File: stack/base-with-services/Dockerfile [hunk 1]
```diff
@@ -25,3 +25,10 @@ WORKDIR "/home/${NB_USER}"
 
 # Initialize the database
 RUN mamba run -n aiida-core-services initdb -D aiida_db -U aiida
+
+# Install Quantum ESPRESSO and its AiiDA plugin
+RUN mamba install qe
+RUN mamba install aiida-quantumespresso==4.0.1
```
Comment (csadorf) [inline stack/base-with-services/Dockerfile:31]: I would suggest to run this in one step:
```suggestion
RUN mamba install \
    aiida-quantumespresso==4.0.1 \
    qe \
  && mamba clean --all -f -y && \
  fix-permissions "${CONDA_DIR}"
```
Also, would be prudent to specify the desired version of `qe`.

File: stack/base/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,14 @@ LABEL maintainer="AiiDAlab Team <aiidalab@materialscloud.org>"
 USER root
 WORKDIR /opt/
 
-ARG AIIDA_VERSION=2.0.0
```
[the remark this answers is not in the document]
Comment (mbercx) [reply stack/base/Dockerfile:9]: So the value specified here is like a backup?

[the remark this answers is not in the document]
Comment (csadorf) [reply stack/base/Dockerfile:9]: It's a default that is known to work that will generally be overwritten by build system. We could consider to remove the default value if it is cause for too much confusion.

[the remark this answers is not in the document]
Comment (mbercx) [reply stack/base/Dockerfile:9]: In what cases would the default actually be used? If it _does_ serve a good purpose we can keep it, but then maybe we can add a comment that this is a default and will be overridden by the value in `build.json`.

[the remark this answers is not in the document]
Comment (csadorf) [reply stack/base/Dockerfile:9]: I originally wrote the image files in a way that you could also build them independently as well (e.g. for debugging), but with the introduction of the [`base` variable](https://github.com/aiidalab/aiidalab-docker-stack/blob/3ffbfc289b33e9f2fafdef48b71f51a5da9d1b6f/stack/base-with-services/Dockerfile#L2) this is no longer the case. So I would be in favor of removing them to reduce potential for confusion and over-specification.

[the remark this answers is not in the document]
Comment (mbercx) [reply stack/base/Dockerfile:9]: I'll open a PR to do this on `main`

[the remark this answers is not in the document]
Comment (csadorf) [reply stack/base/Dockerfile:9]: Resolved with #311 .

**Pull request closed without merging**